### PR TITLE
strain: update tests

### PR DESCRIPTION
**Pull request by robphoenix**
see #414

### CODE REVIEW

**Comment by robphoenix** [comment]
further info in commit messages

File: exercises/strain/strain_test.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ func TestDiscardInts(t *testing.T) {
 		}
 		switch res := cp.Discard(test.pred); {
 		case !reflect.DeepEqual(cp, test.list):
-			t.Fatalf("Ints%v.Discard() should not modify its reciever.  "+
-				"Found %v, reciever should stay %v",
+			t.Fatalf("Ints%v.Discard() should not modify its receiver.  "+
```
Comment (ferhatelmas) [inline exercises/strain/strain_test.go]: What about here ? Because you changed formatting of strings below.

Comment (robphoenix) [inline exercises/strain/strain_test.go]: I was too focused on the strings, I totally overlooked this. Thanks @ferhatelmas

**Comment by robphoenix** [comment]
I've updated the PR after @ferhatelmas review

**Pull request merged**